docs/advanced-features/middleware.md

```diff
@@ -88,6 +88,22 @@ export const config = {
 }
 ```
 
+You can also create a "negative matcher" that matches all paths except for certain paths:
```

Comment:
Should we expand this to say full regex matching can be used including negative lookaheads like this